package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "bulma": "^0.1.2",
```

Comment:
@jbailey4 ^0.1.2 := >=0.1.2 < 1.0.0. Was everything working with the latest bulma?